<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flow Classifier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Testing</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>290</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Simulation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Type of Liquid</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>701</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Custom</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>133</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>org.flowrate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>20</y>
        <width>133</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>pitch</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>L</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pitch">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>40</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_L">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_flowrate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>total_flow_rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>341</x>
        <y>20</y>
        <width>133</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>ratio</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_ratio">
      <property name="geometry">
       <rect>
        <x>341</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_total_flow_rate">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox_experiment">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiments</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>701</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Method</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>681</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_method">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox_type_liquid">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_res">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>711</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>701</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Liquid Values</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>681</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_liquid_vals"/>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Training</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiments for Training</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_experiment_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_training">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Training</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setTestPath">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>find</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_testPath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Test data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Train data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_trainPath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setTrainPath">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>find</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>Command</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionTrain"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuCommand"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRun">
   <property name="text">
    <string>Run Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>Training (with .csv)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
